securedrop/translations/cs/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,1399 @@
+# Translations template for SecureDrop.
+# Copyright (C) 2019 Freedom of the Press Foundation
+# This file is distributed under the same license as the SecureDrop project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: SecureDrop 0.14.0~rc1\n"
+"Report-Msgid-Bugs-To: securedrop@freedom.press\n"
+"PO-Revision-Date: 2019-09-16 23:39+0000\n"
+"Last-Translator: michaela-bot <michaela.rybickova@gmail.com>\n"
+"Language-Team: Czech <https://weblate.securedrop.org/projects/securedrop/"
+"securedrop/cs/>\n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+"X-Generator: Weblate 3.7.1\n"
+"Generated-By: Babel 2.5.1\n"
+
+#: template_filters.py:14
+msgid "{time} ago"
+msgstr "před {time}"
+
+#: journalist_app/__init__.py:88 journalist_app/__init__.py:133
+msgid "You have been logged out due to inactivity"
+msgstr "Z důvodu neaktivity jste byli odhlášeni"
+
+#: journalist_app/__init__.py:142
+msgid "You have been logged out due to password change"
+msgstr "Z důvodu změny hesla jste byl/a odhlášen/a"
```

Comment:
hmm why these "/a"s? is that a special character?

Reply:
I believe (not speaking Czech at all) that this is subject-verb gender agreement when you don't know who "you" is: _byl_ is "he was", _byla_ "she was"; _odhlášen_ is masculine, _odhlášena_ feminine. I'll double-check with the translator.

Reply:
Confirmed.